### PR TITLE
[1LP][RFR] fix: test_delete_datastore_appear_after_refresh

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- added vm and host count property in Model page
- removed some `version.pick`
- It will fix test_delete_datastore_appear_after_refresh

{{py.test: -vvv cfme/tests/infrastructure/test_delete_infra_object.py -k 'test_delete_datastore_appear_after_refresh' --use-provider vsphere55}}

### CODE REVIEW

File: cfme/tests/infrastructure/test_delete_infra_object.py [hunk 1]
```diff
@@ -16,7 +14,7 @@
 ]
 
 
-def test_delete_cluster_appear_after_refresh(setup_provider, provider, appliance):
+def test_delete_cluster_appear_after_refresh(provider, appliance):
```
Comment (quarckster) [inline cfme/tests/infrastructure/test_delete_infra_object.py]: **QUESTION**
Why did you delete `setup_provider`?

Comment (digitronik) [reply cfme/tests/infrastructure/test_delete_infra_object.py]: @quarckster No need of it; What's your opinion?
I think as we are marking `pytest.mark.provider` now no need to go with `setup_provider`.

Comment (digitronik) [reply cfme/tests/infrastructure/test_delete_infra_object.py]: my bad; forgot to add a fixture `pytest.mark.usefixtures('setup_provider')` @quarckster done
